index.html

```diff
@@ -2570,6 +2603,143 @@ <h3>
           </pre>
         </div>
       </section>
+      <section>
+        <h2><a id="icon-masks"></a>
+          Icon masks and safe zone
+        </h2>
+        <p>
+          Some platforms have their own preferred icon shape, but as web
+          applications should work across multiple platforms, it is possible to
+          indicate that an icon may have a user-agent-specified mask applied by
```

Comment:
I'd prefer to use non-normative language in this para i.e. s/may/can/g and list normative statement in the list below.